api/base/utils.py

```diff
@@ -49,11 +49,18 @@ def is_bulk_request(request):
     content_type = request.content_type
     return 'ext=bulk' in content_type
 
+
 def is_truthy(value):
-    return value in TRUTHY
+    if isinstance(value, bool) or value is None:
+        return value
+    return str(value).lower() in TRUTHY
```

Comment:
I think this should be able to handle `float` 1 but will double check before/during merge.